sass/patternfly-react/_patternfly-react.scss

```diff
@@ -1,3 +1,7 @@
 /**
   Patternfly React Partials
 */
+.breadcrumbs-pf-title .active span {
+  font-size: 16px;
+  vertical-align: middle;
+}
```

Comment:
Sorry, updates to the [CONTRIBUTING.MD](https://github.com/patternfly/patternfly-react/blob/master/CONTRIBUTING.md) file had not yet been merged. It has been merged now. Please see the  [Adding Styling for your Components](https://github.com/patternfly/patternfly-react/blob/master/CONTRIBUTING.md#adding-styling-for-your-components) section.